brainglobe_atlasapi/atlas_generation/atlas_scripts/blackcap.py

```diff
@@ -0,0 +1,221 @@
+__version__ = "0"
+
+import csv
+import glob as glob
+import os
+import time
+from pathlib import Path
+
+import numpy as np
+from brainglobe_utils.IO.image import load_nii
+from rich.progress import track
+
+from brainglobe_atlasapi.atlas_generation.annotation_utils import (
+    read_itk_labels,
+)
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+
+def create_atlas(working_dir, resolution):
+    ATLAS_NAME = "oldenburg_blackcap"
+    SPECIES = "Sylvia atricapilla"
+    ATLAS_LINK = "https://uol.de/en/ibu/animal-navigation"
+    CITATION = "unpublished"
+    ATLAS_FILE_URL = "https://uol.de/en/ibu/animal-navigation"  # noqa
+    ORIENTATION = "asr"
+    ROOT_ID = 999
+    ATLAS_PACKAGER = "BrainGlobe Developers, hello@brainglobe.info"
+    ADDITIONAL_METADATA = {}
+
+    # setup folder for downloading
+
+    atlas_path = Path(
+        "/media/ceph/margrie/sweiler/AnalyzedData/"
```

Comment:
Does this atlas fail our "normal" requirement that all data is available online, and the script should run without access to any specific filesystem?

Should we create a blackcap repo on GIN and upload these files?

Reply:
Done: https://gin.g-node.org/BrainGlobe/blackcap_materials